packages/edit-navigation/src/components/layout/style.scss

```diff
@@ -7,3 +7,15 @@
 		padding-left: 0;
 	}
 }
+
+.edit-navigation-layout {
+	display: flex;
+	flex-direction: column;
+	// Make the navigation layout full-height, accounting for wp-admin top bar and other furniture.
+	height: calc(100vh - 146px);
```

Comment:
do we need to use hardcoded values here?

Reply:
I think yes because of the other surrounding stuff from WP Admin which are fixed in size.

Reply:
Have removed those after following @tellthemachines' suggestion